CMakeLists.txt

```diff
@@ -1232,20 +1122,6 @@ endif ()
 ## 3RD PARTY
 ################################################################################
 
-# 3rdParty exports:
-#
-# V8_VERSION
-# V8_LIBS
-# V8_INCLUDE_DIR
-#
-# ICU_VERSION
-# ICU_LIBS
-# ICU_INCLUDE_DIR
-#
-# ZLIB_VERSION
-# ZLIB_LIBS
-# ZLIB_INCLUDE_DIR
```

Comment:
If this comment is accurate (don't know if it is), then I would vote for leaving it there.
This is because the stuff in 3rdParty populates these variables in the parent scope, and it is otherwise not immediately visible that this happens.

Reply:
These are only partially accurate; the ccmake tool can give better insight into what's defined and whats not;

I will look into the necessity of exporting values to the parent scope at all and hopefully find beter ways to deal wit hthis.